package.json

```diff
@@ -9,45 +9,20 @@
     "build": "tsc 1>&2 && npm run copy:test-assets",
     "clean": "rimraf lib out",
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
-    "start": "(npm run build) & node ./lib/main.js",
-    "test": "mocha ./lib/tests/*.test.js",
-    "copy:test-assets": "cpx \"./src/tests/assets/**/{*,.*}\" ./lib/tests/assets"
+    "start": "(npm run build) & node ./lib/main.js"
   },
   "dependencies": {
-    "@bentley/backend-itwin-client": "^2.17.2",
-    "@bentley/bentleyjs-core": "^2.17.2",
-    "@bentley/ecschema-metadata": "^2.17.2",
-    "@bentley/electron-manager": "^2.17.2",
-    "@bentley/frontend-authorization-client": "^2.17.2",
-    "@bentley/geometry-core": "^2.17.2",
-    "@bentley/imodelhub-client": "^2.17.2",
-    "@bentley/imodeljs-backend": "^2.17.2",
-    "@bentley/imodeljs-common": "^2.17.2",
-    "@bentley/imodeljs-frontend": "^2.17.2",
-    "@bentley/imodeljs-i18n": "^2.17.2",
-    "@bentley/imodeljs-quantity": "^2.17.2",
-    "@bentley/itwin-client": "^2.17.2",
-    "@bentley/orbitgt-core": "^2.17.2",
-    "@bentley/presentation-common": "^2.17.2",
-    "@bentley/product-settings-client": "^2.17.2",
-    "@bentley/rbac-client": "^2.17.2",
-    "@bentley/telemetry-client": "^2.17.2",
-    "@bentley/ui-abstract": "^2.17.2",
-    "@bentley/webgl-compatibility": "^2.17.2",
-    "electron": "^11.1.0"
+    "@itwin/insights-client": "0.4.0",
+    "@itwin/core-bentley": "^3.6.1",
+    "@itwin/core-backend": "^3.6.1",
+    "@itwin/node-cli-authorization": "^0.10.0"
   },
   "devDependencies": {
-    "@bentley/build-tools": "^2.17.2",
-    "@bentley/eslint-plugin": "^2.17.2",
-    "@types/chai": "^4.2.21",
-    "@types/mocha": "^8.2.3",
-    "@types/sinon": "^10.0.2",
-    "chai": "^4.3.4",
-    "eslint": "^6.8.0",
-    "mocha": "^9.0.2",
+    "@itwin/build-tools": "3.6.1",
+    "@itwin/eslint-plugin": "3.6.1",
+    "eslint": "^7.11.0",
     "rimraf": "^3.0.2",
-    "sinon": "^11.1.1",
-    "typescript": "^3.7.7"
+    "typescript": "~4.3.0"
   },
   "eslintConfig": {
```

Comment:
`eslintConfig` looks outdated. Update according to https://www.npmjs.com/package/@itwin/eslint-plugin readme.

Reply:
Done.

Reply:
I guess we can close this PR?